dipy/tracking/streamline.py

```diff
@@ -688,7 +683,7 @@ def values_from_volume(data, streamlines, affine=None):
 
     affine : ndarray, shape (4, 4)
         Affine transformation from voxels (image coordinates) to streamlines.
-        Default: identity. For example, if no affine is provided and the first
+        For example, if no affine is provided and the first
```

Comment:
`if no affine is provided`  I think this docstring is irrelevant. `affine` parameter is mandatory so it is impossible to have no affine now